editor/store/effects.js

```diff
@@ -118,12 +118,23 @@ export default {
 
 		let request;
 		if ( isAutosave ) {
-			toSend.parent = post.id;
+			const autosaveData = {};
```

Comment:
Minor: If we use `let` instead of `const` to declare `toSend`, with spread syntax we could simplify this a bit:

```js
toSend = {
	title: post.title,
	excerpt: post.excerpt,
	...toSend,
	parent: post.id
};
```

Reply:
And if we want to take it a step further with [Lodash's `_.pick`](https://lodash.com/docs/4.17.10#pick):

```js
toSend = {
	...pick( post, [ 'title', 'excerpt' ] ),
	...toSend,
	parent: post.id
};
```